src/clj/puppetlabs/services/jruby/jruby_internal.clj

```diff
@@ -1,16 +1,16 @@
-(ns puppetlabs.services.jruby.jruby-puppet-internal
+(ns puppetlabs.services.jruby.jruby-internal
   (:require [schema.core :as schema]
-            [puppetlabs.services.jruby.jruby-puppet-schemas :as jruby-schemas]
+            [puppetlabs.services.jruby.jruby-schemas :as jruby-schemas]
             [clojure.tools.logging :as log]
             [puppetlabs.kitchensink.core :as ks])
-  (:import (com.puppetlabs.puppetserver.pool JRubyPool)
-           (puppetlabs.services.jruby.jruby_puppet_schemas JRubyPuppetInstance PoisonPill ShutdownPoisonPill)
+  (:import (com.puppetlabs.jrubyutils.pool JRubyPool)
```

Comment:
I swear I tried that and IntelliJ complained at me, but it seems like it must have been complaining for a different reason, because that totally works. Agreed that that's more consistent and an easy change.